src/Bicep.Core.Samples/Files/InvalidModules_LF/main.diagnostics.bicep

```diff
@@ -545,7 +545,7 @@ module wrongModuleParameterInLoop2 'modulea.bicep' = [for (x,i) in emptyArray:{
     stringParamA: 'test'
     stringParamB: 'test'
     notAThing: 'test'
-//@[4:13) [BCP037 (Error)] No other properties are allowed on objects of type "params". |notAThing|
```

Comment:
>allowed [](http://example.com/codeflow?start=51&length=7)

We might be losing coverage on the original error here. Could you undo the change to modulea and add a similar moduleb for your test instead?

Reply:
Fixed.